blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md

```diff
@@ -0,0 +1,300 @@
+---
+title: How to Tokenize a Physical Asset on Celo
+description: Understand how to start the tokenization on Celo
+authors:
+- name: Ewerton Lopes
+  title: Founder of Guizo Studios
+  url: https://github.com/guizostudios
+  image_url: https://avatars.githubusercontent.com/u/97633823
+tags: [celo, solidity, developers, celosage]
+hide_table_of_contents: false
+slug: /tutorials/how-to-tokenize-a-physical-asset-on-celo
+---
+
+
+![header](../../src/data-tutorials/showcase/beginner/how-to-tokenize.png)
+
+## What is tokenization?
+
+Tokenization is the process of creating a digital token that represents a real-world asset. This token can be traded on a blockchain, which is a distributed ledger that allows for the secure and transparent transfer of ownership of the asset.
+
+Tokenization has the **potential to revolutionize** the way we own and trade assets by making it easier to buy, sell, and transfer ownership of a wide range of assets, including real estate, precious metals, and even fine art. It can also help to increase liquidity and reduce the costs associated with trading assets, as the use of smart contracts and blockchain technology can streamline the transaction process.
+
+On July 14, 2022, Itaú, a big bank in Brazil, launched its own cryptocurrency unit, which aims to tokenize fixed income securities and shares traded in the market in order to make alternatives that are currently restricted to high net worth investors more accessible.
```

Comment:
On July 14, 2022, Itaú, a big bank in Brazil, launched its cryptocurrency unit, which aims to tokenize fixed-income securities and shares traded in the market to make alternatives that are currently restricted to high-net-worth investors more accessible.

Reply:
Done